Formula/trojan-go.rb

```diff
@@ -0,0 +1,105 @@
+class TrojanGo < Formula
+  desc "Trojan proxy in Go"
+  homepage "https://p4gefau1t.github.io/trojan-go/"
+  url "https://github.com/p4gefau1t/trojan-go/archive/v0.10.0.tar.gz"
+  sha256 "423fa05d81da0105a01682a77bbbd95fef09644c3a9dbbddd641436e31879429"
```

Comment:
Change `url` and `sha256` to this convention: https://github.com/Homebrew/homebrew-core/blob/1220c789207176204e85bb597fe56530da4f503c/Formula/glooctl.rb#L4-L6

Because building Trojan-Go needs the `version` and `commit` variable passed to go build and test.

Reply:
Roger that. Thanks.